.circleci/config.yml

```diff
@@ -1,14 +1,19 @@
-version: 2
+version: 2.1
+orbs:
+  cypress: cypress-io/cypress@1.8.0
 jobs:
-  build:
-    docker:
-      - image: mamolli/cypress_awscli
+  lint_and_unit_test:
     working_directory: ~/repo
+    docker:
+      - image: circleci/node:10.16.1
     steps:
       - checkout
+      - restore_cache:
+          keys:
+            - node-v1-{{ .Branch }}-{{ checksum "package-lock.json" }}
```

Comment:
why not just checksum?

Reply:
Followed [docs](https://circleci.com/docs/2.0/caching/) in this case. 

>If your source code changes frequently, we recommend using fewer, more specific keys. This produces a more granular source cache that will update more often as the current branch and git revision change.

```   
steps:
  - restore_cache:
      keys:
        # when lock file changes, use increasingly general patterns to restore cache
        - node-v1-{{ .Branch }}-{{ checksum "package-lock.json" }}
        - node-v1-{{ .Branch }}-
        - node-v1-

```